packages/forms/src/UIForm/fieldsets/Array/Array.component.test.js

```diff
@@ -243,6 +243,40 @@ describe('Array component', () => {
 			expect(props.onChange).toHaveBeenCalledWith(expect.anything(), payload);
 			expect(props.onFinish).toHaveBeenCalledWith(expect.anything(), payload);
 		});
+
+		it('should add default value from schema as default value for new item', async () => {
+			const defaultValue = {
+				name: 'default name',
+				email: 'default email',
+				comment: 'default comment',
+			};
+
+			const enhancedSchema = {
+				...schema,
+				schema: {
+					...schema.schema,
+					default: [defaultValue],
```

Comment:
this looks like a default value for the entire array, not only when adding an item

Reply:
I used default value defined to initialize component with one single value to create new entry with same default value.
So here the default value for first element and that will be used for next added ones.